chat-widget/src/components/livechatwidget/common/endChat.ts

```diff
@@ -55,8 +55,13 @@ const prepareEndChat = async (props: ILiveChatWidgetProps, chatSDK: any, setAdap
     if (isPostChatEnabled === "true" && conversationDetails?.canRenderPostChat === Constants.truePascal) {
         const skipEndChatSDK = false;
         const skipCloseChat = true;
+        const chatSession = await chatSDK?.getCurrentLiveChatContext();
         await endChat(props, chatSDK, setAdapter, setWebChatStyles, dispatch, adapter, skipEndChatSDK, skipCloseChat, false);
-
+        if (chatSession) {
+            (chatSDK as any).chatToken = chatSession.chatToken ?? {};
```

Comment:
**Minor**: Please disable eslint warning for any

Reply:
done